5187 jwt refresh token Feature (#5589)
strongloop/loopback-next: extensions/authentication-jwt/README.md

  }
```

> ### End points with refresh token

Reviewer: ```suggestion
### Endpoints with refresh token
```

> To add refresh token mechanism in your app. you have to call the following

Reviewer: ```suggestion
To add refresh token mechanism in your app, you have to call the following
```

> interrceptor.

Reviewer: typo:
```suggestion
interceptor.
```

> 1. ('refresh-token-generate') : to generate the refresh token and access token

Reviewer: Maybe using this style instead to be consistent with the rest of the documentation?

```suggestion
1. `refresh-token-generate` : to generate the refresh token and access token
```

   when user logins to your app with provided credentials.

```ts
 @intercept('refresh-token-generate')
async refreshLogin(
  @requestBody(CredentialsRequestBody) credentials: Credentials,
): Promise<TokenObject> {
  // ensure the user exists, and the password is correct
  const user = await this.userService.verifyCredentials(credentials);
  // convert a User object into a UserProfile object (reduced set of properties)
  const userProfile: UserProfile = this.userService.convertToUserProfile(
    user,
  );
  // create a JSON Web Token based on the user profile
  const token = {
    accessToken: await this.jwtService.generateToken(userProfile),
  };
  return token;
}
```

> 2. ('refresh-token-grant'): to genearate the access token by the refresh token

Reviewer: Suggestion on style change and fix typo: 
```suggestion
2. `refresh-token-grant`: to generate the access token by the refresh token
```

>    obtained from the the last login end-point.

Reviewer: ```suggestion
   obtained from the the last login endpoint.
```
